src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala

```diff
@@ -43,87 +57,137 @@ case class GetManyDocumentsQueryBuilder(
           """.stripMargin)
   }
 
-  def conflicts(conflicts: Boolean = true): GetManyDocumentsQueryBuilder = {
-    set("conflicts", conflicts)
+  def conflicts(
+      conflicts: Boolean = true): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("conflicts", conflicts)
+  }
+
+  def descending(
+      descending: Boolean = true): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("descending", descending)
+  }
+
+  def endKey[K: W](endKey: K): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("endkey", write(endKey))
+  }
+
+  def endKeyDocId(
+      endKeyDocId: String): GetManyDocumentsQueryBuilder[ID, AM, BT] = {
+    set[ID, AM, BT]("endkey_docid", endKeyDocId)
+  }
+
+  def includeDocs[D: R]: GetManyDocumentsQueryBuilder[IncludeDocs[D], AM, BT] = {
```

Comment:
In that case, maybe add an overload similar to `allowMissing`? I would have to have an `if` statement and return `IncludeDocs[D]` or `ExcludeDocs` based on the argument. The reason I asked for this is that I tried keeping the method names match the parameter names in the CouchDB API, this makes it easier for the users to find the method. With the rest of the setters you can just pass a `false` to disable the option. If we don't have it here, the users would have to guess that they need to call `excludeDocs` instead of `includeDocs(false)` (which is like it's done for the other options). However, I'm not sure if it's really necessary (having multiple ways to do the same thing is not very good either), what do you think? Maybe to keep it consistent, remove `excludeDocs`?